browser/IgBlazorSamples.Client/wwwroot/index.html

```diff
@@ -43,6 +43,9 @@
     <script src="./js/2.index.bundle.js"></script>
     <script src="./js/3.index.bundle.js"></script>
     <script type="module" src="./js/index.bundle.js"></script>
+	<!-- scripts for Excel -->
+	<script src="_framework/blazor.webassembly.js"></script>
+    <script src="_content/IgniteUI.Blazor.Documents.excel/excel.js"></script>
```

Comment:
move this line after:
`<script src="_content/IgniteUI.Blazor/app.bundle.js"></script>`